src/Compilers/CSharp/Test/Semantic/Semantics/RefFieldTests.cs

```diff
@@ -22323,6 +22323,300 @@ public void ScopedReserved_Alias_Escaped()
                 );
         }
 
+        [Fact, WorkItem("https://github.com/dotnet/roslyn/issues/75828")]
+        public void UnscopedRefAttribute_RefSafetyRules_ThisParameter()
+        {
+            var source = """
+                using System.Diagnostics.CodeAnalysis;
+                struct S
+                {
+                    public int F;
+
+                    [UnscopedRef] public ref int Ref() => ref F;
+                }
+                """;
+            CreateCompilation([source, UnscopedRefAttributeDefinition],
+                parseOptions: TestOptions.Regular10).VerifyDiagnostics(
+                // (6,6): error CS9063: UnscopedRefAttribute cannot be applied to this parameter because it is unscoped by default.
```

Comment:
>error CS9063: UnscopedRefAttribute cannot be applied to this parameter because it is unscoped by default. [](http://example.com/codeflow?start=26&length=105)

This doesn't feel like the right error message for earlier language versions because `this` is not unscoped by default, as indicated by the next error: "error CS8170: Struct members cannot return 'this' or other instance members by reference".

Similarly, for the following, when compiled with C#10 and .NET 6, we're reporting that the parameter (in this case, `ref int i`) is unscoped by default. But that also seems surprising since callers can assume the `ref` is not captured in the `Span<int>` when `UseUpdatedEscapeRules` is `false`.
```csharp
static Span<int> F([UnscopedRef] ref int i) { ... }
```

Instead, should we report a diagnostic that `[UnscopedRef]` is not supported with C#10 and .NET 6 and earlier? Should this be a warning rather than an error so it's not a breaking change?